src/resources/views/manual/tutorials/qgis-getting-started.blade.php

```diff
@@ -0,0 +1,69 @@
+@extends('manual.base')
+
+@section('manual-title', 'Getting Started')
+
+@section('manual-content')
+  <div class="row">
+    <p class="lead">Introduction on Interfacing Biigle Api with QGis.</p>
```

Comment:
Please use consistent acronyms (e.g. API instead of Api, BIIGLE instead of Biigle, QGIS instead of QGis) everywhere.